src/shared/ui/short-info-panel/short-info-panel.tsx

```diff
@@ -0,0 +1,64 @@
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+
+import { DateSimple } from '../date-simple';
+import micIcon from '@/shared/assets/icons/mic.svg';
+import { LABELS } from '@/shared/constants';
+import { calculateFreshDate } from '@/shared/helpers/getCourseDate';
+import { Language } from '@/shared/types';
+
+import styles from './short-info-panel.module.scss';
+
+const cx = classNames.bind(styles);
+
+interface ShortInfoPanelProps {
+  startDate: string | null;
+  registrationEndDate: string | null;
+  mode?: string;
+  language: Language;
+  withMargin?: boolean;
+  onlyLanguage?: boolean;
+  label?: string | null;
+}
+
+export const ShortInfoPanel = ({
+  startDate,
+  registrationEndDate,
+  language,
+  mode,
+  withMargin,
+  onlyLanguage,
+  label = LABELS.START_DATE,
```

Comment:
What do you think about removing the default label value and updating the type to `label?: string`?
That way, we could just add `label = {LABELS.START_DATE}` in one spot, and not have to pass `label={null}` everywhere else